recipes-core/fs-init/files/resizefs.sh

```diff
@@ -12,37 +12,59 @@ if [ -z $2 ]; then
     echo "Missing in parameter to target PART ($1pX)"
     exit 1
 fi
-
 DISK="$1"
 PART="$2"
 
-logger "resizing ${DISK}p$PART to fill its full partition size"
-# get the disk total size
-#DISK_SIZE=`cat /sys/block/$DISK/size`
+if grep "/$DISK" /proc/cmdline; then
 
-# get partition start and size
-PART_OFF=$(cat /sys/block/$DISK/${DISK}p$PART/start)
-#PART_SIZE=`cat /sys/block/$DISK/${DISK}p$PART/size`
+    logger "resizing ${DISK}p$PART to fill its full partition size"
+    # get the disk total size
+    #DISK_SIZE=`cat /sys/block/$DISK/size`
 
-# Resize now
-if [ -f "/opt/hm/mx4_env" ]; then
-    #For mx4 use one less \n before $PART_OFF for some strnge reason. 
-    #if the mx4_env is not there we assume machines like Mxv.
-    printf "d\n$PART\nn\np\n$PART_OFF\n\nw\n" | fdisk -B  /dev/${DISK}
-else
-    printf "d\n$PART\nn\np\n\n$PART_OFF\n\nw\n" | fdisk -B  /dev/${DISK}
-fi
+    # get partition start and size
+    PART_OFF=$(cat /sys/block/$DISK/${DISK}p$PART/start)
+    #For only variant 1 hmx that use config parition better to move to seperate distro..
+    if grep "HM010-1-" /proc/device-tree/chosen/part-number; then
```

Comment:
@mattiasbus, I realize this is wrong part number this is not PCBA it is the product part number!